contracts/plugins/governance/Governance.sol

```diff
@@ -30,7 +31,9 @@ contract Governance is
     // 100%
     uint256 public constant ONE_HUNDRED_PERCENT = 1e8; // {micro %}
 
-    // solhint-disable no-empty-blocks
+    // solhint-disable-next-line var-name-mixedcase
+    uint256 public immutable MIN_VOTING_DELAY; // in blocks, based on network
```

Comment:
let's reference here in the comment that it should be equal to `ONE_DAY`, or whatever we end up calling the 86400 figure as a constant

Reply:
will do the changes and push

Reply:
also can use our unit notation `{blocknumber}` here